clients/web/config/default.json

```diff
@@ -0,0 +1,13 @@
+{
+    "client": {
+        "gqlUrl": "",
+        "secure": false,
+        "chatwoot": {
+            "enabled": false,
+            "url": "",
+            "websiteToken": ""
+        }
+    },
+    
+    "server": {}
```

Comment:
Should a value exist here?

Reply:
No, just a placeholder for config we want to add in the future